docs/application/web/guides/data/file-system.md

```diff
@@ -108,149 +107,261 @@ To manage file storages:
+
+   Perform all actual operations, such as reading, writing, or closing on the file through the `FileHandle` object based on a current position, which can be obtained or modified through calling `seek()` operation on `FileHandle` object.
 
    ```
-   var documentsDir;
+   /* Opening file for write - file is created if not exists, */
+   /* otherwise existing file is truncated */
+   var fileHandleWrite = tizen.filesystem.openFile('documents/file', 'w');
+   console.log('File opened for writing');
+   ```
 
-   tizen.filesystem.resolve('documents', function(result) {
-       documentsDir = result;
-   });
+   - use writeString() or writeStringNonBlocking() to write `DOMString` to file.
+   ```
+   var stringToWrite = 'example string';
+   fileHandleWrite.writeString(stringToWrite);
+   ```
 
-   var testFile = documentsDir.createFile('test.txt');
-   if (testFile != null) {
-       testFile.openStream('rw', onOpenSuccess, null, 'UTF-8');
-   }
+   - use writeData() or writeDataNonBlocking() to write `Uint8Array` object contents to file.
+   ```
+   var arrayToWrite = new Uint8Array([11,22,88,99]);
+   fileHandleWrite.writeData(arrayToWrite);
    ```
 
-2. Perform all actual operations, such as reading, writing, or closing, on the file through the `FileStream` object based on a position attribute, which represents the current position in the file:`function onOpenSuccess(fs) {    /* Write HelloWorld to the file */    fs.write('HelloWorld');    /* Move pointer to the beginning */    fs.position = 0;    /* Read the file content from the beginning */    fs.read(testFile.fileSize);    /* Close the file */    fs.close();}`
+   - use writeBlob() or writeBlobNonBlocking() to write `Blob` object contents to file.
+   ```
+   var blobToWrite = new Blob(['example blob content']);
+   fileHandleWrite.writeBlob(blobToWrite);
+   ```
 
-3. To copy a file or directory, use the `copyTo()` method. The following example copies the files to the `images/backup/` directory. Since the third parameter is set to `true`, any existing files with the same name in the target directory are overwritten.
+   - when file is no longer needed, remember to close it
+   ```
+   fileHandleRead.close();
+   ```
+ - You can copy, move and rename files and directories within the file system with the `copyFile()`, `copyDirectory()`, `moveFile()`, `moveDirectory()` and `rename()` methods. During copy or move operations, if a file or directory of the same name already exists in the target location, the overwrite input parameter of the method defines whether the existing file is overwritten.
+
+1. To copy a file, use the `copyFile()` method. The following example copies the file to the `images` directory. Since the third parameter is set to `true`, any existing file or directory with the same name in the target directory is overwritten.
```

Comment:
directory is overwritten:

Use a ":" at the end of the sentence as we have used the word following.

Reply:
Done.